api/client/beacon/health_test.go

```diff
@@ -1,112 +0,0 @@
-package beacon
```

Comment:
Deleting all tests for the health tracker? Shouldn't there be some tests?

Reply:
Two reasons:
- there is barely any logic
- the update takes place every 12 seconds, so the test would be super slow (we could make the interval a parameter, but it would be only for testing, therefore I am not a big fan of this)